src/components/pages/Categories/Categories.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div
+    :class="categoryObject"
```

Comment:
[imo]

条件によってクラスが付与されるか否かというのは今回必要ないと思うので、そういった時はバインドさせる必要はなく、単純に以下のように記述することができます。

また、`is-〇〇` というふうにクラス名を設定してもらってると思いますが、`is` という単語はプログラミング界隈では、真偽値を表す時に使用されます。
上述した通り、今回は特に class を付与するということに対して条件はないので、単純に `category` などのクラス名で良いと思います。

他の箇所も同じような修正が必要そうなので修正の方お願いします。

```suggestion
    class="category"
```

Reply:
通常の記述方法でclassを記述しました。